imdclient/IMDClient.py

```diff
@@ -237,6 +249,17 @@ def _go(self):
         self._conn.sendall(go)
         logger.debug("IMDClient: Sent go packet to server")
 
+    def _wait(self, IMDwait=None):
```

Comment:
Capitalization of first letter is typically reserved for class names, can you change this to "imdwait"?